R/linter_tags.R

```diff
@@ -58,7 +58,8 @@ available_linters <- function(packages = "lintr", tags = NULL, exclude_tags = "d
   }
 
   # any tags specified explicitly will not be excluded (#1959)
-  exclude_tags <- setdiff(exclude_tags, tags)
+  # never include defunct linters, which don't work / error on instantiation (#2284).
```

Comment:
This undocumented behavior might surprise. Why not add it to the default `exclude_tags` and let users query which linters are defunct?

Reply:
That will break the workflow `available_linters(tags = NULL)`. I'll just document it instead.

Reply:
Which one? Our own test?

Reply:
It's really not worth spending more effort supporting a way to access defunct linters. They're defunct.